PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp/Info.plist

```diff
@@ -36,24 +45,10 @@
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
-		<string>UIInterfaceOrientationLandscapeLeft</string>
```

Comment:
I'm not sure if this is correct approach. Does this affect the video fullscreen in landscape mode? Can't this be moved only to the view that you need and not the whole app?